doc/cephadm/services/oauth2-proxy.rst

```diff
@@ -126,7 +126,7 @@ The `oauth2-proxy` service typically uses the default container image:
 
 ::
 
-    DEFAULT_OAUTH2_PROXY = 'quay.io/oauth2-proxy/oauth2-proxy:v7.2.0'
+    DEFAULT_OAUTH2_PROXY_IMAGE = 'quay.io/oauth2-proxy/oauth2-proxy:v7.2.0'
```

Comment:
The tag is still incorrect here. It's very easy to miss updating this ( ... see https://version.ninja/), that's why I recommend taking it out.

Reply:
ah, I didn't even look at the image, just the variable. I think I'll replace this with a note telling the command to get whatever the current image is rather than this hardcoded bit

Reply:
I redid this section as described in my other comment above